meson.build

```diff
@@ -22,6 +22,8 @@ localedir = join_paths(prefix, get_option('localedir'))
 includedir = join_paths(prefix, get_option('includedir'))
 localstatedir = join_paths(prefix, get_option('localstatedir'))
 sysconfdir = join_paths(prefix, get_option('sysconfdir'))
+pkglibdir = join_paths(libdir, meson.project_name()) # TODO: -1 suffix? soname?
```

Comment:
Any suffix would be for the purposes of parallel-installability, and hence would have to include the major version of the project, and the major version of the plugin API. Those versions might be linked so that only one version number needs to be included.

I would recommend adding in a major version number suffix here, yes.